### ISSUE TITLE
Path configuration is confusing and inconsistent with frontend PeerJS client config

### ISSUE THREAD

**Issue by intellix** [peers/peerjs-server#234]
### I have a suggestion:
Currently the suggestion is to do something like the following:
```
const peerServer = ExpressPeerServer(server);
app.use('/peerjs', peerServer);
```

This actually creates a binding on `/peerjs/peerjs` because inside the ExpressPeerServer middleware it's adding an `app.use(options.path, xyz)`.

Alongside that, on the client there isn't double paths without being explicit:
```ts
const peer = new Peer('123', {
  host: 'localhost',
  port: 9001,
  path: '/peerjs/peerjs',
  secure: false,
  debug: 2,
  config: {
    iceServers,
  },
});
```

**Comment by spine001** [comment]
You are correct, this is a big source of confusion, but changing the code simply to correct that would affect all the working code out there. I have found that most of the tutorials, if not all don't actually work. Often due to changes in the code. My suggestion is to improve the manual! I didn't know of this double app.use(....) that you mentioned here, and thanks to you now I do. I solved this by explicitly doing:

```
const peerServer = ExpressPeerServer(mainServer, {
        debug: true,
        path: '',
        ssl: {},
})
```

**Comment by yalcinozer** [comment]
Yeah, it is confusing to me also. The client socket request Url contains "peerjs", which is something I want to customize. It adds it with default settings.
